packages/pigeon/example/app/windows/runner/flutter_window.cpp

```diff
@@ -49,15 +75,6 @@ bool FlutterWindow::OnCreate() {
     return false;
   }
 
-  // #docregion cpp-method-flutter
-  void TestPlugin::CallFlutterMethod(
```

Comment:
This code was not fully C++, and was in the wrong place (as in, this isn't a valid place for this definition to exist in C++). I pulled it out into a class to match Swift and Kotlin, and updated it to compile.